gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 109
+cordaApiRevision = 110
```

Comment:
Just curious why this was required? Is there a breaking change here?

Reply:
The problem is that if we won't explicitly change the number if someone will pull the latest release branch and have an old cache the build will fail.